docs/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1783 @@
+{
```

Comment:
<p>Maybe simplify to:</p><p> </p><p><strong>Remark:</strong> Again, note that <code>aesara</code> is clever enough to omit the <code>exp</code> and <code>log</code> once we compile the function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>+1</p>

Reply:
+1